packages/labs/ssr-dom-shim/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "@lit-labs/ssr-dom-shim",
+  "version": "1.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "DOM shim for Lit Server Side Rendering (SSR)",
+  "license": "BSD-3-Clause",
+  "author": "Google LLC",
+  "homepage": "https://github.com/lit/lit/tree/main/packages/labs/ssr-dom-shim",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lit/lit.git",
+    "directory": "packages/labs/ssr-dom-shim"
+  },
+  "main": "index.js",
+  "typings": "index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "default": "./index.js"
```

I believe we need to add types here too.
```suggestion
      "types": "./index.d.ts",
      "default": "./index.js"
```

---

Done